wic/src/main/java/edu/asu/diging/wic/core/service/IConceptTextService.java

```diff
@@ -0,0 +1,53 @@
+package edu.asu.diging.wic.core.service;
+
+import java.util.List;
+
+import edu.asu.diging.wic.core.model.impl.ConceptText;
+
+public interface IConceptTextService {
+
+    /**
+     * <p>Method is used to add conceptText model/object in database
+     * </p>
+     * @param conceptText Model of conceptText which will be stored 
+     */
+    void addText(ConceptText conceptText, String name);
+
+    /**
+     * <p>Method is used to find all conceptText to be shown on a particular page
```

Comment:
remove "to be shown on a particular page". It could be used for anything not just showing on a page.